### PR TITLE
replace jrevels with that sleek new maintainer ararslan

### PR DESCRIPTION
**Pull request by jrevels**
cc @ararslan 

After this is merged, we should update the cluster's local Nanosoldier.jl repos (on each node, in each Julia package directory) and restart the server. Doing that will be my last act as Nanosoldier's primary maintainer 🔨

### CODE REVIEW

File: bin/setup_test_ci.jl [hunk 1]
```diff
@@ -8,8 +8,8 @@ secret = ENV["GITHUB_SECRET"]
 
 config = Nanosoldier.Config(ENV["USER"], nodes, cpus, auth, secret;
                             workdir = joinpath(homedir(), "test_workdir"),
-                            trackrepo = "jrevels/julia",
-                            reportrepo = "jrevels/BaseBenchmarkReports",
+                            trackrepo = "ararslan/julia",
+                            reportrepo = "ararslan/BaseBenchmarkReports",
```
Comment (ararslan) [inline bin/setup_test_ci.jl]: I guess this means I should fork BaseBenchmarkReports

Comment (jrevels) [inline bin/setup_test_ci.jl]: You only need to if you run the test CI

**Comment by StefanKarpinski** [comment]
Why is anyone's name hardcoded in this? Baking in one person's account seems like a bad practice.

**Comment by jrevels** [comment]
I found it really useful when I was maintaining Nanosoldier to get all the Nanosoldier-related traffic. Main point of this PR was to remove my account and signal to others who the new maintainer was.

Of course, if Alex wants to change this, he's an owner of the repo and can do so.

**Pull request merged**